Assets/Source/Entities/Behaviors/Commander.cs

```diff
@@ -7,9 +7,11 @@ public class Commander : MonoBehaviour
 {
 	// Get rid of these after testing movement
 	private GameObject tempCircle;
+	private Player player;
 
 	void Start()
 	{
+		player = transform.root.GetComponent<Player>();
```

Comment:
Actually, there's some interesting behavior for this command which may prove annoying later if we're not careful.
`transform.root` actually finds the parent as far up as it can. This means that if we were to move all a player's units into a unit tree under the player object, the player object would always be selected by transform.root.

If we were to use location aware parent searching (aka. I am in this component and I need to go up one and then over down two game objects to get the component I want), our code would be super unflexible, but I'm not sure of a better way to do things. It'd be nice if we could keep all the components for a unit in one game object. But that may not be possible.

An alternate solution might be to include a reference to the 'central' game object in each unit object. That way we could use things like `centralObject.GetComponentInChildren<ArbitraryComponent>();`.